doc/source/examples/analysis/alignment_and_rms/README.rst

```diff
@@ -10,10 +10,10 @@ The :mod:`MDAnalysis.analysis.align` and :mod:`MDAnalysis.analysis.rms`
 modules contain the functions used for aligning structures,
 aligning trajectories, and calculating root mean squared quantities.
 
-Demonstrations of alignment are in `align_structure`_,
-`align_trajectory_first`_, and `align_trajectory`_. Another example of
+Demonstrations of alignment are in `align_structure <aligning_structure_to_another.ipynb>`__,
+`align_trajectory_first <aligning_trajectory_to_first_frame.ipynb>`__, and `align_trajectory <aligning_trajectory.ipynb>`__. Another example of
```

Comment:
change `aligning_trajectory_to_first_frame.ipynb` to `aligning_trajectory_to_frame.ipynb` and double check that the link resolves